### PR TITLE
Stale Issues

### PR DESCRIPTION
**Pull request by Sjoerd-Bo3**
Stale issues will get Stale notifications after 30 days and closed after 14 days
Stale issues that 'needs-triage' will get Stale notifications after 30 days and closed after 30 days

### CODE REVIEW

**Comment by marionbarker** [comment]
This got used with Loop and it cleaned up the backlog, but you need an option to mark issues as longer-term items that need to stay open.

**Comment by Sjoerd-Bo3** [comment]
> This got used with Loop and it cleaned up the backlog, but you need an option to mark issues as longer-term items that need to stay open.

Good idea! Will implement! Also the featurerequests will not be on GH in the future.

File: .github/workflows/stale_triage_issues.yml
Comment (bjornoleh) [inline .github/workflows/stale_triage_issues.yml:1]: Is this empty file needed? 😀

Comment (Sjoerd-Bo3) [reply .github/workflows/stale_triage_issues.yml:1]: Whoops, no ofc not. Opted to use 1 file instead of 2

**Review by bjornoleh — APPROVED** [review]

**Review by bjornoleh — APPROVED** [review]

**Review by AndreasStokholm — APPROVED** [review]
👏

**Comment by bjornoleh** [comment]
Merging with two approvals

**Pull request merged**